packages/dashboard/src/layout/useLayout.ts

```diff
@@ -0,0 +1,18 @@
+import { useContext } from 'react';
+import GoldenLayout from '@deephaven/golden-layout';
+import LayoutContext from './LayoutContext';
+
+/**
+ * Retrieve the current layout from the context
+ */
+function useLayout(): GoldenLayout {
```

Comment:
Calling this `layout` has always been a little confusing to me. I wonder if we should call it `useGoldenLayout` instead. I think there are some other spots where `layout` is the object that specifies the actual layout vs the instance of golden layout (often `glLayout` or something like that)

Or `useLayoutManager` which I think is what GoldenLayout calls the class